doc/source/cookbook.rst

```diff
@@ -1034,7 +1036,7 @@ You can use the same approach to read all files matching a pattern.  Here is an
 Finally, this strategy will work with the other ``pd.read_*(...)`` functions described in the :ref:`io docs<io>`.
 
 .. ipython:: python
-    :suppress:
+    :verbatim:
```

Comment:
What does `:verbatim:` does? I assume the suppress made those not appear in the rendered docs, is that still the case?

Reply:
:verbatim: prevents execution as well. In this case I think I have made a mistake. The decorator should remain suppress.